pyproject.toml

```
@@ -0,0 +1,73 @@
+[build-system]
+requires = ["setuptools>=64.0", "setuptools_scm[toml]"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pydm-converter-tool"
+description = "converter to generate pydm .ui files from other display managers."
+readme = "README.md"
+authors = [
+  { name = "Devan Agrawal"},
+  { name = "Zach Domke"},
+  { name = "Ryan McClanahan"},
+  { name = "Alexander Ng"},
+  { name = "Yekta Yazar", email = "yazar@slac.stanford.edu"}
+]
+keywords = ["PyDM", "converter", "GUI"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "License :: OSI Approved :: GNU General Public License (GPL)",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering",
+]
+requires-python = ">=3.11"
+dependencies = [
+    "pydm",
+    'pyqt5',
+]
+dynamic = ["version"]
+
+# [tool.setuptools_scm]
+# version_file = "src/_version.py"
+
+[project.optional-dependencies]
+dev = [
+    "pytest",
+    "pytest-cov",
+    "pytest-qt",
+    "pytest-mock"
+]
+
+[project.urls]
+Homepage = "https://github.com/slaclab/pydm-converter-tool"
+# Documentation = ""
+"Bug Tracker" = "https://github.com/slaclab/pydm-converter-tool/issues"
+
+[project.scripts]
+badger = "badger.__main__:main"
+
+[project.license]
+file = "LICENSE"
+
+[options]
+zip_safe = false
+include_package_data = true
+
+# tool.setuptools.packages.find]
+# where = ["src"]
+# include = [ "", ]
+# namespaces = false
+
+[tool.ruff.lint]
+extend-select = ["TID252"] # Defaults + check imports
```

Might want to add a line length to ruff so we aren't limited to 80 chars:
```suggestion
extend-select = ["TID252"] # Defaults + check imports
line-length = 120
```